test/string_view-test/string_view-test.cpp

```diff
@@ -0,0 +1,343 @@
+/*
+* string_view-test.cpp
+* Sean Murthy, Ray Chen
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Test string_view template
+* - see C++17 [string.view]
+*	https://timsong-cpp.github.io/cppwp/n4659/string.view
+*/
+
+#include <stdexcept>
+
+#include "../verifiers.h"
+
+void string_view_test()
+{
+	using std::string_view;
+
+	//1. Construction and assignment
+	//2. Capacity
+
+	// Default constructor. Constructs an empty string_view. 
+	string_view empty_sv;
+	is_zero(empty_sv.size(), "empty_sv.size()");
+	is_true(empty_sv.length() == empty_sv.size(), "empty_sv.length() = empty_sv.size()");
+	is_true(empty_sv.empty(), "empty_sv.empty()");
+
+	//constructs a string_view with const char
+	string_view sv("hello, world!");
+	is_true(sv.size() == 13, "sv.size()");
+	is_true(sv.length() == sv.size(), "sv.length() = sv.size()");
+	is_false(sv.empty(), "!sv.empty()");
+
+	//Constructs a view of the first 5 characters of the character array
+	string_view f_sv("hello, world!", 5);
+	is_true(f_sv.size() == 5, "f_sv.size()");
+	is_true(f_sv.length() == f_sv.size(), "f_sv.length() = f_sv.size()");
+	is_false(f_sv.empty(), "!f_sv.empty()");
+
+	//Copy constructor. Constructs a view of the same content as other.
```

Comment:
Keep comments simple

```suggestion
	//copy constructor
```